package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@fuel-infrastructure/fuel-hyperlane-registry",
   "description": "A collection of configs, artifacts, and schemas for Hyperlane",
-  "version": "0.0.2",
+  "version": "0.0.1main",
```

should be 0.0.1-main?

```suggestion
  "version": "0.0.1-main",
```

---

that was my first attempt but caused error :)

---

I see ok